src/components/ContextMenuItem.tsx

```diff
@@ -93,7 +108,7 @@ function ContextMenuItem(
             title={itemText}
             icon={itemIcon}
             onPress={triggerPressAndUpdateSuccess}
-            wrapperStyle={styles.pr9}
+            wrapperStyle={[wrapperStyle, styles.pr9]}
```

Comment:
```suggestion
            wrapperStyle={[styles.pr9, wrapperStyle]}
```

In case we want to override this `pr9` style.